tests/dialects/test_bigquery.py

```diff
@@ -1013,45 +1017,20 @@ def test_bigquery(self):
             },
         )
         self.validate_all(
-            "SELECT cola, colb FROM (VALUES (1, 'test')) AS tab(cola, colb)",
-            write={
-                "spark": "SELECT cola, colb FROM VALUES (1, 'test') AS tab(cola, colb)",
+            "SELECT cola, colb FROM UNNEST([STRUCT(1 AS cola, 'test' AS colb)])",
+            read={
```

Comment:
These tests were changed, previous ones were incorrect in both that the input queries were invalid in BQ and the expected queries were again invalid BQ.